tools/docker/roles.sql

```diff
@@ -1 +1 @@
-CREATE ROLE awx;
+CREATE ROLE awx SUPERVISOR LOGIN PASSWORD 'awx';
```

Comment:
Change SUPERVISOR to SUPERUSER?
```suggestion
CREATE ROLE awx SUPERUSER LOGIN PASSWORD 'awx';
```